src/framework/theme/components/calendar-kit/components/calendar-day-picker/calendar-day-cell.component.spec.ts

```diff
@@ -30,6 +30,7 @@ describe('Component: NbCalendarDayCell', () => {
   });
 
   it('should contain cell class', () => {
+    fixture.detectChanges();
```

Comment:
>  I'm dead sure we need it because of moving class declaration from host param to HostBinding decorator

Right.

> it might be we don't need to move that class declaration to the HostBinding

As discussed above, decorator metadata doesn't inherit. If we keep host class there, styling would break. I believe it's worse than having dumb `cssClass = true` binding.